slick.grid.js

```diff
@@ -6262,11 +6324,9 @@ if (typeof Slick === "undefined") {
       "getCanvasWidth": getCanvasWidth,
       "getCanvases": getCanvases,
       "getActiveCanvasNode": getActiveCanvasNode,
-      "setActiveCanvasNode": setActiveCanvasNode,
       "getViewportNode": getViewportNode,
       "getViewports": getViewports,
       "getActiveViewportNode": getActiveViewportNode,
-      "setActiveViewportNode": setActiveViewportNode,
```

Comment:
isn't there a way to keep all existing functions? That could be a breaking for a few users, probably me included with my libs, so I would rather keep these 2 functions if possible